src/environments/environment.ts

```diff
@@ -14,5 +14,7 @@ export const environment = {
   uPlanetCode: 'guatemala',
   uParentCode: 'guatemala@earth',
   upgradeAddress: window.location.origin + '/upgrade',
-  syncAddress: window.location.protocol + '//localhost:5984'
+  syncAddress: window.location.protocol + '//localhost:5984',
+  mobileBreakpoint: 780,
```

Comment:
is there reason for using it as env? It could be constant if it does not change for each deployment

Reply:
No reason for that. Ok